assets/sass/modules/_mfa-challenge-forms.scss

```diff
@@ -71,3 +71,16 @@
     width: 100%;
   }
 }
+
+//idx pipeline
+.mfa-verify-idxpipeline {
```

Comment:
Can we not use `idxpipeline` in our variable names  and come up with something more generic ?